logstash-core/spec/logstash/api/modules/node_stats_spec.rb

```diff
@@ -98,26 +98,11 @@
          "queue_push_duration_in_millis" => Numeric
        },
        "flow" => {
-         "output_throughput" => {
-           "lifetime" => Numeric,
-           "current" => Numeric
-         },
-         "filter_throughput" => {
-           "lifetime" => Numeric,
-           "current" => Numeric
-         },
-         "queue_backpressure" => {
-           "lifetime" => Numeric,
-           "current" => Numeric
-         },
-         "worker_concurrency" => {
-           "lifetime" => Numeric,
-           "current" => Numeric
-         },
-         "input_throughput" => {
-           "lifetime" => Numeric,
-           "current" => Numeric
-         }
+         "output_throughput" => Hash,
+         "filter_throughput" => Hash,
+         "queue_backpressure" => Hash,
+         "worker_concurrency" => Hash,
+         "input_throughput" => Hash
```

Comment:
This is something I am stuck now. Unless we capture the flow metric, `current` & `lifetime` metric values are not stored. [Test logic](https://github.com/elastic/logstash/blob/1fa8535742e183c1f460f550b3c25b7998b0477d/spec/support/resource_dsl_methods.rb#L44) uses `last_response.body` which doesn't contain the current and lifetime. I am digging up and trying to figure out _how I can capture metrics so that reflects into  `last_response.body`_. That's the reason my PR is a draft.

Reply:
Ah. Actually _running_ the periodic poller for long enough to capture meaningful flow rates is outside the scope of this test.

Reply:
As we discussed and decided I have removed flow metric from `PipelinesInfo` DS. It is used in periodic metric snapshot capture purpose but not involved in our APIs.
- I think we are pretty well made a unit test confidence by confirming the API structure contains flow metrics (`stats_specs.rb:128L~162L`, `metrics_spec.rb:243L`).
- Also integration tests (now running, I will chase after failures if happens) to check flow metrics (`monitoring_api_spec.rb:248L`) and make sure they are reset when pipeline is reloaded (`reload_config_spec.rb:88L`).